src/main/java/uk/gov/hmcts/darts/common/entity/EventHandlerEntity.java

```diff
@@ -38,11 +41,18 @@ public class EventHandlerEntity {
     @Column(name = "handler")
     private String handler;
 
+    @Column(name = "active")
```

Comment:
Wondering, should we be specifying `nullable = false` on the columns that are now showing `NOT NULL` in the reference script. I think so as it makes it clearer and consistent, looking at the entity class.

Reply:
done